test/jdk/tools/launcher/DisableBestFitMappingTest.java

```diff
@@ -23,9 +23,10 @@
 
 /*
  * @test
- * @bug 8337506
- * @summary Verify Command Line arguments are not mapped with
- *     "best-fit" mappings on Windows
+ * @bug 8337506 8349254
+ * @summary Verify command line arguments, including ones from
+ *          "JDK_JAVA_OPTIONS" environment variables are not mapped
+ *          with "best-fit" mappings on Windows
  * @requires (os.family == "windows")
  * @library /test/lib
  * @run junit DisableBestFitMappingTest
```

Comment:
I think it might be best to re-write this test as a non Junit test. Through IntelliJ it's hard to run this test, I presume because of the combination of it being a Junit test and having a `main` method? If I run the 'launcher' suite of tests, this test does not seem to be included.

Reply:
I am not sure of this. Isn't it the issue in jtreg plugin? At leas `make test TEST=...` will succeed.

Reply:
Oops, I thought the test would mark as "skipped" or something like that, but yes its a Windows only test, hence why it does not show up, duh.